contracts/lib/lifecycle/Destructible.sol

```diff
@@ -1,9 +1,14 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
 
 import "../ownership/Ownable.sol";
 
+
+//was not compiling with selfdestruct(owner) added a require and changed parameter from owner to msg.sender
+
+
 contract Destructible is Ownable {
     function selfDestruct() public onlyOwner {
-        selfdestruct(owner);
+        require(owner == msg.sender);
+        selfdestruct(msg.sender);
```

Comment:
//was not compiling with selfdestruct(owner) added a require and changed parameter from owner to msg.sender